Lib/test/test_generated_cases.py

```diff
@@ -253,6 +253,23 @@ def run_cases_test(self, input: str, expected: str):
                 lines.pop(0)
             while lines and lines[-1].startswith(("#", "\n")):
                 lines.pop(-1)
+            while lines and tier1_generator.INSTRUCTION_START_MARKER not in lines[0]:
```

Comment:
You can replace this line by line processing, including the start and end comment stripping, by splitting the whole file on `INSTRUCTION_START_MARKER` and `INSTRUCTION_START_MARKER`, discarding the first and last parts.

```Py
text = temp_output.read()
_, rest = text.split(INSTRUCTION_START_MARKER)
actual, _ = rest.split(LABEL_START_MARKER)
```

Reply:
Good observation. Thanks!